openc3-s3/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG OPENC3_DEPENDENCY_REGISTRY=docker.io
+ARG ALPINE_VERSION=3.22
+ARG ALPINE_BUILD=2
+ARG OPENC3_VERSITYGW_VERSION=1.0.20
+
+FROM ${OPENC3_DEPENDENCY_REGISTRY}/alpine:${ALPINE_VERSION}.${ALPINE_BUILD}
+
+ARG OPENC3_VERSITYGW_VERSION
+ARG TARGETARCH
+
+# Copy and extract versitygw binary from local tar.gz files
+COPY versitygw_v${OPENC3_VERSITYGW_VERSION}*.tar.gz /tmp/
+RUN ARCH=$([ "$TARGETARCH" = "arm64" ] && echo "arm64" || echo "x86_64") && \
+    tar xzf /tmp/versitygw_v${OPENC3_VERSITYGW_VERSION}_Linux_${ARCH}.tar.gz -C /tmp versitygw_v${OPENC3_VERSITYGW_VERSION}_Linux_${ARCH}/versitygw && \
+    mv /tmp/versitygw_v${OPENC3_VERSITYGW_VERSION}_Linux_${ARCH}/versitygw /usr/local/bin/ && \
+    chmod +x /usr/local/bin/versitygw && \
+    rm -rf /tmp/versitygw_*
```

Comment:
This install uses the released binaries from the release assets page: https://github.com/versity/versitygw/releases/tag/v1.0.10. We'll have to manually grab updates  ... probably needs to be in the package_audit script.